packages/mobile/src/screens/signon/SignOn.tsx

```diff
@@ -339,8 +339,6 @@ const SignOn = ({ navigation }: SignOnProps) => {
 
   const isKeyboardOpen = useSelector(getIsKeyboardOpen)
 
-  // TODO: Remove when fully on react native store
-  const dappLoaded = useSelector(getDappLoaded)
   const lifecycleSignIn = useSelector(getIsSignedIn)
```

Comment:
We can probably get this from the account slice instead

Reply:
so thinking we don't even need right? this screen will only load when "notSignedIn" is true in AppScreen, at which point we can immediately run the sign-on screen animations since we have libs ready to go right away?

Reply:
Oh yup makes sense